functions/pull.go

```diff
@@ -79,6 +84,8 @@ func Pull(restart bool) (models.HostPull, bool, bool, error) {
 	if restart {
 		logger.Log(3, "restarting daemon")
 		return models.HostPull{}, resetInterface, replacePeers, daemon.Restart()
+	} else {
+		_ = wireguard.SetPeers(replacePeers)
```

Comment:
why are we setting peers here?

Reply:
Without it, some of the new addded peers will be missed in the wg peers as tested.

Reply:
this will be a duplicate call, we already set peers outside this func

Reply:
commit added.